samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/delivery/BeginDeliveryActionListener.java

```diff
@@ -101,11 +101,12 @@ public void processAction(ActionEvent ae) throws
       // e.g. take assessment via url, actionString is set by LoginServlet.
       // preview and take assessment is set by the parameter in the jsp pages
       delivery.setActionString(actionString);
-      if ("reviewAssessment".equals(actionString) || "takeAssessment".equals(actionString)) {
-        delivery.setRbcsToken(rubricsService.generateJsonWebToken(RubricsConstants.RBCS_TOOL_SAMIGO));
-      }
     }
-    
+
+    if ("reviewAssessment".equals(delivery.getActionString()) || "takeAssessment".equals(delivery.getActionString()) || "takeAssessmentViaUrl".equals(delivery.getActionString())) {
```

Comment:
StringUtils.equalsAny, I know this pattern is already used in the file but equalsAny is more desirable.